### PR TITLE
Add ne and sw to bounds object

### PR DESCRIPTION
**Pull request by lukyanovfedor**
Hi, thanks for the great library it serving to me faithfully not first time. 

I found small problem with bounds object, most geo tools expect to receive data with NE and SW, instead of NW and SE. So i thought it will be cool if bounds object will contain all corners.

### CODE REVIEW

**Comment by istarkov** [comment]
Hi thank you, fully agree. Will check PR later today or tomorrow.

**Comment by istarkov** [comment]
All is fine, may be add ability to use NE SW (if NW SE is not provided) to documented [fitBounds](https://github.com/istarkov/google-map-react/blob/7ae63cb3d733b91ef14332ed52dd38b5886459f5/src/utils/utils.js#L67) ?

**Comment by lukyanovfedor** [comment]
@istarkov It's make sense, what do you think about such approach: 

1) If we have NW and SE we're just using them.
2) If we have NE and SW we're converting them into NW and SE and using same algorithm that we have in `fitBounds` method.

**Comment by istarkov** [comment]
Super!

**Comment by lukyanovfedor** [comment]
@istarkov cool, i'll push changes tomorrow morning

**Comment by istarkov** [comment]
Thank you!

**Comment by lukyanovfedor** [comment]
@istarkov hm, I thought a little bit about it today, and i don't really like that we're passing NE and SW to `fitBounds`, but it returns NW and SE. What do you think about it? 

Also i think will can add some helpers like `convertNeSwToNwSe` and `convertNwSeToNeSw`, cause we're repeating this logic in few places.

**Comment by istarkov** [comment]
Agree that newBounds must have all corners,
any your decision on `convertXXX2YYY` will be good.

**Comment by lukyanovfedor** [comment]
@istarkov done!

File: src/utils/utils.js [hunk 1]
```diff
@@ -63,54 +63,73 @@ function meters2WorldSize(meters, { lat, lng }) {
...
+    center: world2LatLng(middle),
+    zoom,
+    newBounds: {
+      nw: newNW,
+      se: newSE,
+    },
+  };
+}
+
+const exports = {
```
Comment (istarkov) [inline src/utils/utils.js]: May be we will remove exports and all that `this` below, as it was time when `export default { blabla }` was the same as `export const blabla`
Having that it was deprecated in babel there is no need in exports object at all.

Comment (lukyanovfedor) [inline src/utils/utils.js]: Oh, that part seemed strange to me) Sure.

Comment (lukyanovfedor) [inline src/utils/utils.js]: Done!

**Comment by istarkov** [comment]
Super, 
from first days I had knowledge that bounds object is not similar to others API 
(and my laziness didn't allow me to fix that)

all that my
```
  const ne = { lat: nw.lat, lng: se.lng };
  const sw = { lat: se.lat, lng: nw.lng };
  return new LatLngBounds(sw, ne);
```
now will gone ;-)

Thank you a lot!!!!

**Comment by istarkov** [comment]
At npm google-map-react@0.22.0

**Comment by lukyanovfedor** [comment]
Cool, glad that i helped 😉

**Pull request merged**